lib/data_element.rb

```diff
@@ -34,6 +34,12 @@ def create(elements)
 
         Dhis2::Status.new(response)
       end
+
     end
+
+      def delete
+          puts "deleting #{@id}"
+          Dhis2.resource["dataElements/#{@id}"].delete {|response, request, result| response.code == 204 }
```

Comment:
remove puts and move the code to the "base.rb" class - will make it usable for all items